toolchain/parser/parse_node_kind.h

```diff
@@ -41,7 +41,7 @@ class ParseNodeKind {
   }
 #include "toolchain/parser/parse_node_kind.def"
 
-  // The default constructor is deleted as objects of this type should always be
+  // The default constructor is deleted as the objects of this type should always be
```

Comment:
Again, maybe my English grammar weakness is showing, but the original seems correct w/o the added "the"?

Reply:
I agree that "objects" alone would also be correct but the addition of "the" is a preferred way.

Reply:
```suggestion
  // The default constructor is deleted because objects of this type should always be
```

Here's a different change that may read better. "as" is something I'd tend to avoid in this case. I do agree with chandlerc that "the" is not needed.